Frontend/ui-customer/src/Screens/CustomerUtils/AboutUsScreen/Controller.purs

```diff
@@ -51,6 +54,9 @@ eval :: Action -> AboutUsScreenState -> Eval Action ScreenOutput AboutUsScreenSt
 
 eval (GenericHeaderActionController (GenericHeaderController.PrefixImgOnClick)) state = continueWithCmd state [do pure BackPressed]
 
-eval BackPressed state = exit $ GoToHomeScreen
+eval BackPressed state = if isParentView FunctionCall then do 
+          _ <- pure $ emitTerminateApp Nothing true
+          continue state
+          else exit $ GoToHomeScreen
```

Comment:
write with proper indentation and use `void` instead if `_ <-` - 

```purescript
eval BackPressed state = 
    if isParentView FunctionCall 
      then do 
          void $ pure $ emitTerminateApp Nothing true
          continue state
       else 
          exit $ GoToHomeScreen
```

Reply:
Can we have indentation like this?

Reply:
It is like this only.
<img width="422" alt="Screenshot 2023-11-24 at 9 01 38 AM" src="https://github.com/nammayatri/nammayatri/assets/113873457/ae36d4ae-1acd-4eec-b8d3-29eb23696282">

Reply:
<img width="445" alt="Screenshot 2023-11-24 at 9 04 34 AM" src="https://github.com/nammayatri/nammayatri/assets/62455315/a05d9fe7-40db-41a1-b9ea-cd4b192fc204">
<img width="489" alt="Screenshot 2023-11-24 at 9 04 52 AM"
<img width="566" alt="Screenshot 2023-11
![Uploading Screenshot 2023-11-24 at 9.05.11 AM.png…]()
-24 at 9 05 02 AM" src="https://github.com/nammayatri/nammayatri/assets/62455315/d7df0a06-308d-448b-ab4f-9df42e74a885">
 src="https://github.com/nammayatri/nammayatri/assets/62455315/042eed28-1700-49db-a77b-53da169a4559">

Reply:
Done.